backend/app/views/spree/admin/shared/_stock_sub_menu.html.erb

```diff
@@ -1,8 +1,8 @@
 <ul id="sub_nav" class="admin-subnav" data-hook="admin_stock_transfer_sub_tabs">
   <% if can? :admin, Spree::StockItem %>
-    <%= tab :stock_items, label: 'management', match_path: '/stock_items' %>
+    <%= tab :stock_items, label: :stock_management, match_path: '/stock_items' %>
   <% end %>
   <% if can? :admin, Spree::StockTransfer %>
-    <%= tab :stock_transfers, label: 'transfers', match_path: '/stock_transfers' %>
+    <%= tab :stock_transfers, label: :stock_transfers, match_path: '/stock_transfers' %>
```

Comment:
Isn't the `label` argument redundant, since the `tab` helper uses the first argument as label anyways?